data/nlu/nlu.md

```diff
@@ -5054,3 +5070,9 @@ data/nlu/lookups/products.txt
 
 ## regex:zipcode
 - [0-9]{5}
+
+## lookup:location.txt
+  data/nlu/lookups/location.txt
+
+## lookup:products.txt
+  data/nlu/lookups/products.txt
```

Comment:
This is annoying and Rasa X shouldn't be doing this IMO 👀not sure if you want to merge it or not (the features are the same regardless of the lookup name right?

Reply:
i mean, Rasa X will continue doing it so I would just merge it, the name doesn't matter